kded/keyboardmodule.h

```diff
@@ -0,0 +1,27 @@
+
+#pragma once
+
+#include <memory>
+#include <QSystemTrayIcon>
+#include <KDEDModule>
+#include <KSharedConfig>
+
+class KeyboardModule : public KDEDModule
+{
+    Q_OBJECT
+    Q_CLASSINFO("D-Bus Interface", "com.valvesoftware.keyboard")
+
+public:
+    explicit KeyboardModule(QObject *parent, const QVariantList &);
+
+    Q_SCRIPTABLE bool isEnabled() const;
+    Q_SCRIPTABLE void setEnabled(bool enabled);
+    Q_SCRIPTABLE Q_SIGNAL void enabledChanged(bool enabled);
```

Comment:
This looks like a Q_PROPERTY :thinking:

Reply:
Yes, I wonder if would make the gtk side nice

Reply:
I'm not sure, it would basically only mean connecting to a different signal. I rather prefer to keep the GTK and Qt code as similar as possible, the GTK code is annoying enough as it is.